README.md

```diff
@@ -233,13 +233,154 @@ mstorage.info("<media_id>", new CompletionHandler<MediaInfo>() {
 });
 ```
 
-### Get media metadata
+### Get all media metadata
 ```java
 mstorage.meta("<media_id>", new CompletionHandler<MediaMeta>() {
     @Override
     public void onCompleted(MediaMeta mediaMeta) {
         Map<String, String> exif = mediaMeta.getExif();
         Map<String, String> gpano = mediaMeta.getGpano();
+        Map<String, String> userMeta = mediaMeta.getUserMeta();
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+### Get specific media metadata
+
+* Exif
+
+```java
+mstorage.exif("<media_id>", new CompletionHandler<Map<String, String>>() {
+    @Override
+    public void onCompleted(Map<String, String> exif) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Google Photo Sphere XMP
+
+```java
+mstorage.gpano("<media_id>", new CompletionHandler<Map<String, String>>() {
+    @Override
+    public void onCompleted(Map<String, String> gpano) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* All user metadata
+
+```java
+mstorage.userMeta("<media_id>", new CompletionHandler<Map<String, String>>() {
+    @Override
+    public void onCompleted(Map<String, String> userMeta) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Specific user metadata
+
+```java
+mstorage.userMeta("<media_id>", "<key>", new CompletionHandler<String>() {
+    @Override
+    public void onCompleted(String value) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+### Attach user metadata to media
+
+```java
+mstorage.addUserMeta("<media_id>", "<key>", "<value>", new CompletionHandler<Object>() {
+    @Override
+    public void onCompleted(Object result) {
+        // Do something.
+        // result may be empty.
```

Comment:
As a sentence, starting with a small letter looks weird.  Also, the object is defined as "result", so it's also bad to change it to "Result". So, I think we have two other options below, or never care about it.
* "result" may be empty.
* The returned object may be empty.